SPEC.md

```diff
@@ -56,11 +56,10 @@ In some ways an ACI can be thought of as equivalent to a static binary.
 ### Image Layout
 
 The on-disk layout of an App Container Image is straightforward.
-It includes a *rootfs* with all of the files that will exist in the root of the app, and an *app image manifest* describing the contents of the image and how to execute the app.
+It includes a *rootfs* directory with all of the files that will exist in the root of the app, and an *app image manifest* file describing the contents of the image and how to execute the app.
 
 ```
 /manifest
-/rootfs
```

Comment:
What's your motivation for omitting this? This arguably conflicts with the second stipulation under "Image Archives" (line 75/74). (I realise we're trying to have it both ways here - by not showing the rootfs/usr/ or rootfs/usr/bin/ directories - but imo seems like it's better to err on the side of verbosity and include them instead of removing this)